_includes/footer.html

```diff
@@ -1,6 +1,6 @@
 <footer class="usa-footer usa-footer-big" role="contentinfo">
   <div class="usa-grid usa-footer-return-to-top">
-    <a href="#">Return to top</a>
+    <a href="#main">Return to top</a>
```

Comment:
This won't actually take the user to the *very* top of the page, it'll take them to the main content that's just after the common nav. To take them to the *very* top we'd have to add a new `id` to something at the very top of the page, but I'm not sure if that's actually what we want, so I'm just linking to `#main` for now.